bears/c_languages/GNUIndentBear.py

```diff
@@ -152,16 +154,16 @@ def create_arguments(filename, file, config_file,
             Uses Kernighan & Ritchie coding style.
         :param linux_style:
             Uses Linux coding style.
-        :param tab_width:
-            Number of spaces per indent level.
+        :param indent_size:
+            Number of spaces representing one tab.
```

Comment:
the point was that couldn't it be better refrained as "The number of spaces per indentation level" ? we are trying to avoid the word `tab` when we are already using spaces right? until necessary i.e.